openassessment/xblock/static/sass/xb/utilities/_reset.scss

```diff
@@ -288,6 +288,14 @@
   }
 
   /**
+   * Set opacity on buttons to 50% (which matches LMS styling).
```

Comment:
I'd rather this use the Pattern Library definition. Does that also use opacity of 50%? The LMS will be switching over to Pattern Library buttons.